pkgs/top-level/python-packages.nix

```diff
@@ -4657,6 +4657,10 @@ in {
 
   seqdiag = callPackage ../development/python-modules/seqdiag { };
 
+  sequoia = toPythonModule (callPackage ../tools/security/sequoia {
```

Comment:
If it is called entirely from `python-packages.nix` then the file should live `python-modules/`.

The way I see it, this package provides bindings like certain other packages do. In that case, call it from `all-packages.nix` and then here in `python-packages.nix` override it to pass in `python`.

Reply:
I didn't yet figured this out. Could you explain this further? Perhaps using another package as an example?

Reply:
See e.g. the attributes `boost`, `caffe` and `libxslt`. Each of them exists in both `all-packages.nix` and `python-packages.nix`.

Reply:
Please see the latest version and my last commit.